README.md

```diff
@@ -3,7 +3,14 @@ The beginnings of a Sculpin-based style guide creation tool.
 
 ## Dependencies
 1. NodeJS 
+   1. __Recommended method__: To intall NodeJS using Homebrew run 'brew install node' 
+   2. To install NodeJS without Homebrew, visit https://nodejs.org/en/download/ and select the correct file download. Follow the instructions here.
```

Comment:
@aCyborg I think considering we're telling people in the (drupal8-skeleton)[https://github.com/palantirnet/drupal8-skeleton] to install ansible with brew we can just include the recommended step without the other here for NodeJS